w3-access.md

```diff
@@ -93,7 +95,9 @@ Provider SHOULD deny service if DID in the `with` field has no storage provider.
 
 #### delegate `nb`
 
-Field is a set of delegations links represented as a JSON. Keys SHOULD be CID strings of the values to make encoding deterministic.
+##### delegate `nb.delegations`
+
+Field is a set of delegation links represented as JSON. Keys SHOULD be CID strings of the values to make encoding deterministic.
```

Comment:
Why is this not an array? e.g. `[{ "/": "bafy...prf1" }]`

I assumed because the delegation could be inlined, but wording suggests otherwise.

Reply:
@alanshaw I think @Gozala originally used a dict to emulate a set and not have to require any sorting to get deterministic encoding.